Fix typo in manual page, needs to be forwarded to uptsream.
---
--- dvisvgm_30.orig/doc/dvisvgm.1
+++ dvisvgm_30/doc/dvisvgm.1
@@ -1461,7 +1461,7 @@
 \fIpdf:pagesize\fR
 is similar to the
 \fIpapersize\fR
-special (see above) which specifies the size of the current and all folowing pages\&. In order to actually apply the extents to the generated SVG files, option
+special (see above) which specifies the size of the current and all following pages\&. In order to actually apply the extents to the generated SVG files, option
 \fB\-\-bbox=papersize\fR
 must be given\&.
 .sp
